tests/libs/test_matrix.py

```diff
@@ -47,3 +56,60 @@ def test_message_from_dict(token_network, get_accounts, get_private_key):
         message_from_dict(message_json)
 
     assert "Invalid message data. Can not find the data type" in str(excinfo.value)
+
+
+def test_deserialize_messages_empty():
+    messages = deserialize_messages(data="", peer_address=PEER_ADDRESS)
+    assert len(messages) == 0
+
+
+def test_deserialize_messages_empty_lines():
+    messages = deserialize_messages(data=" \r\n ", peer_address=PEER_ADDRESS)
+    assert len(messages) == 0
+
+
+def test_deserialize_messages_invalid_json():
+    messages = deserialize_messages(data="@@@", peer_address=PEER_ADDRESS)
+    assert len(messages) == 0
+
+
+def test_deserialize_messages_invalid_message_type(request_monitoring_message):
+    message_json = request_monitoring_message.to_dict()
+    message_json["type"] = "SomeNonexistantMessage"
+
+    messages = deserialize_messages(data=json.dumps(message_json), peer_address=PEER_ADDRESS)
+    assert len(messages) == 0
+
+
+def test_deserialize_messages_invalid_message_class(request_monitoring_message):
+    message_json = request_monitoring_message.to_dict()
+
+    with patch("raiden_libs.matrix.message_from_dict", new=Mock()):
+        messages = deserialize_messages(data=json.dumps(message_json), peer_address=PEER_ADDRESS)
+        assert len(messages) == 0
+
+
+def test_deserialize_messages_invalid_sender(request_monitoring_message):
+    message_json = request_monitoring_message.to_dict()
+
+    messages = deserialize_messages(data=json.dumps(message_json), peer_address=PEER_ADDRESS)
```

Comment:
Just from the variable name, I would have expected `PEER_ADDRESS` to be the correct `peer_address`.